README.md

```diff
@@ -1,11 +1,11 @@
 # splunk-forwarder-operator
 
-This operator manages Splunk Universal Forwarder. It deploys a daemonset which 
+This operator manages [Splunk Universal Forwarder](https://docs.splunk.com/Documentation/Forwarder/8.0.2/Forwarder/Abouttheuniversalforwarder). It deploys a daemonset which 
```

Comment:
Can you link to https://docs.splunk.com/Documentation/Forwarder/latest/Forwarder/Abouttheuniversalforwarder instead?

Reply:
@wshearn Changed the link, as pointed